airbyte-integrations/connectors/destination-bigquery-denormalized/build.gradle

```diff
@@ -31,3 +31,9 @@ dependencies {
 
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
 }
+
+configurations.all {
+  resolutionStrategy {
+    force 'com.google.api-client:google-api-client:1.31.5'
```

Comment:
Was this the version of `google-api-client` we were using before the updated versions or the version most compatible with all versions `2.17.X` and `2.22.X`

Reply:
just what we had previously. I didn't want to spend even more time validating different versions to find the latest compatible one